boards/xtensa/m5stack_atoms3/m5stack_atoms3.yaml

```diff
@@ -0,0 +1,21 @@
+identifier: m5stack_atoms3
+name: M5Stack AtomS3
+type: mcu
+arch: xtensa
+toolchain:
+  - zephyr
+supported:
+  - gpio
+  - i2c
+  - spi
+  - watchdog
+  - regulator
+  - uart
+  - pinmux
+  - nvs
+  - display
+testing:
+  ignore_tags:
+    - net
+    - bluetooth
+vendor: espressif
```

Comment:
@kartben, nitpick by @LucasTambor : vendor here should be `m5stack` instead.

Reply:
not a nitpick at all - thanks! I will update all other m5stack boards accordingly.